### PR TITLE
app-admin/hardinfo: add 0.6_alpha_pre20201116

### PR DESCRIPTION
**Pull request by jsmolic**
Fix building with gcc-10. Thanks to Peter Alfredsen for submitting the
ebuild.

Closes: https://bugs.gentoo.org/752054
Closes: https://bugs.gentoo.org/707722

### CODE REVIEW

File: app-admin/hardinfo/hardinfo-0.6_alpha_pre20201116.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+DESCRIPTION="System information and benchmark tool for Linux systems"
+HOMEPAGE="https://github.com/lpereira/hardinfo"
+SRC_URI="https://github.com/lpereira/hardinfo/archive/${GIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gtk2 debug"
+
+RDEPEND="gtk2? ( >=x11-libs/gtk+-2.10:2 )
```
Comment (SoapGentoo) [inline app-admin/hardinfo/hardinfo-0.6_alpha_pre20201116.ebuild:19]: consider dropping this and just forcing GTK 3

File: app-admin/hardinfo/hardinfo-0.6_alpha_pre20201116.ebuild [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	>=dev-libs/glib-2.10:2
+	>=net-libs/libsoup-2.42:2.4
+	sys-libs/zlib"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+S="${WORKDIR}/${PN}-${GIT_COMMIT}"
+
+src_configure() {
+	use debug && append-flags -fsanitize=address
```
Comment (SoapGentoo) [inline app-admin/hardinfo/hardinfo-0.6_alpha_pre20201116.ebuild:34]: drop this, too fancy

File: app-admin/hardinfo/hardinfo-0.6_alpha_pre20201116.ebuild [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	mycmakeargs=(
```
Comment (SoapGentoo) [inline app-admin/hardinfo/hardinfo-0.6_alpha_pre20201116.ebuild:35]: `local`

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: app-admin/hardinfo/hardinfo-0.6_alpha_pre20201116.ebuild (hunk 1 above)
Comment (jsmolic) [reply app-admin/hardinfo/hardinfo-0.6_alpha_pre20201116.ebuild]: Done

File: app-admin/hardinfo/hardinfo-0.6_alpha_pre20201116.ebuild (hunk 2 above)
Comment (jsmolic) [reply app-admin/hardinfo/hardinfo-0.6_alpha_pre20201116.ebuild]: Dropped

File: app-admin/hardinfo/hardinfo-0.6_alpha_pre20201116.ebuild (hunk 3 above)
Comment (jsmolic) [reply app-admin/hardinfo/hardinfo-0.6_alpha_pre20201116.ebuild]: Done

**Review by juippis — APPROVED** [review]
```
 * QA Notice: new icons were found installed but icon cache
 * has not been updated:
 *   /usr/share/icons/hicolor/48x48/apps/hardinfo.png
 * Please make sure to call xdg_icon_cache_update()
 * in pkg_postinst() and pkg_postrm() phases of appropriate pkgs.
>>> app-admin/hardinfo-0.6_alpha_pre20201116 merged.
```
Will add, thanks.

**Pull request closed without merging**